core/src/epicli/data/common/ansible/playbooks/roles/haproxy_exporter/tasks/main.yml

```diff
@@ -26,21 +25,19 @@
     file_name: "{{ exporter_file_name }}"
 
 - name: Create /opt/haproxy_exporter directories
-  become: yes
   file:
     path: "{{ item }}"
-    recurse: yes
+    recurse: true
```

Comment:
Is there a common practice of boolean values usage? Seems like not necessary change (this one and others)

Reply:
Please take a look at this issue https://github.com/epiphany-platform/epiphany/issues/1296
Currently support for yes/no/on/off... is completely broken for the python part of Epiphany :-1:
YAML 1.2 dropped support of the alternative forms https://yaml.org/spec/1.2/spec.html#tag/repository/bool
Also IMHO the mere idea of having something like `on` for `true` and `no` for `false` is ridiculous :)

I don't know if these are good reasons, I like to think that they are ;)

Reply:
The second one seems to be reasonable.